runtime_builders/builder_util.py

```diff
@@ -169,6 +78,18 @@ def load_manifest_file():
         os.remove(tmp)
 
 
+def file_exists(remote_path):
+    try:
+        logging.info('Checking file {0}'.format(remote_path))
+        tmpdir = tempfile.mkdtemp()
+        tmp_file = os.path.join(tmpdir, 'tmp')
```

Comment:
There's a 'gsutil stat' or a 'gsutil ls' command that wouldn't require copying the file locally.

Reply:
updated